### PR TITLE
Task: drop mapi version and download the CLI from app.mayhem.security

### PR DESCRIPTION
**Pull request by james-kessler**
Completes https://forallsecure.atlassian.net/browse/MH-13526

### CODE REVIEW

**Comment by james-kessler** [comment]
Your development orb has been published. It will expire in 30 days.
You can preview what this will look like on the CircleCI Orb Registry at the following link: 
https://circleci.com/developer/orbs/orb/forallsecure/mapi?version=dev:1bbdeb88cf411d396e105cf5e75ae88213ef1d6c

**Comment by james-kessler** [comment]
Your development orb has been published. It will expire in 30 days.
You can preview what this will look like on the CircleCI Orb Registry at the following link: 
https://circleci.com/developer/orbs/orb/forallsecure/mapi?version=dev:1581a0e5f8541751a7b9cb0cea1183b592e11b92

File: .circleci/test-deploy.yml [hunk 1]
```diff
@@ -19,6 +19,7 @@ jobs:
       - mapi/scan:
           api-url: "https://demo-api.mayhem4api.forallsecure.com/api/v3"
           api-spec: "https://demo-api.mayhem4api.forallsecure.com/api/v3/openapi.json"
+          target: "forallsecure/mapi-circleci-orb"
```
Comment (james-kessler) [inline .circleci/test-deploy.yml]: @swarkentin How should this be specified now?  I am getting a 403 in my tests with `forallsecure/mapi-circleci-orb`

Comment (swarkentin) [reply .circleci/test-deploy.yml]: To target a workspace, use the fully qualified path of:

`workspace/project/target`

So... how about:

`forallsecure/integrations/mapi-circleci-orb`

?

**Comment by james-kessler** [comment]
Your development orb has been published. It will expire in 30 days.
You can preview what this will look like on the CircleCI Orb Registry at the following link: 
https://circleci.com/developer/orbs/orb/forallsecure/mapi?version=dev:8247978a710f1c27793171c738c1744aa753a7a5

File: .circleci/test-deploy.yml (hunk 1 above)
Comment (james-kessler) [reply .circleci/test-deploy.yml]: thank you

**Review by Ross-ForAllSecure — APPROVED** [review]

**Review by swarkentin — APPROVED** [review]

**Comment by james-kessler** [comment]
Your development orb has been published. It will expire in 30 days.
You can preview what this will look like on the CircleCI Orb Registry at the following link: 
https://circleci.com/developer/orbs/orb/forallsecure/mapi?version=dev:e6f33a391320a2e03b2a049c57476ed09ca469a1

**Pull request merged**